docs/pages/includes/enterprise/ent-user-prereqs.mdx

```diff
@@ -0,0 +1,31 @@
+<Tabs>
+<TabItem
+  scope={["enterprise"]} label="Self-Hosted">
+
+- Access to the machine hosting the Auth Service
```

Comment:
Should we mention that you can also manage a self-hosted cluster remotely with `tctl`? This might be easier for the user if accessing the machine hosting the Auth Service is difficult. Or does this require elevated permissions that you can only obtain by running `tctl` on the same machine as the Auth Service?